tools/PI/DevHome.PI/Strings/en-us/Resources.resw

```diff
@@ -654,16 +654,16 @@
     <comment>Tooltip for the system memory usage box in the bar</comment>
   </data>
   <data name="PIDisplayName" xml:space="preserve">

```

Comment:
>PI [](http://example.com/codeflow?start=14&length=2)

A little more messy, but these should probably be renamed DIDisplayName....

Reply:
yeah will be in my next PR, going to change the namespace, directory, executable name, etc. but I anted to keep this one purely aesthetic so it wouldn't interrupt anyone's pending PRs.